gettsim/config.py

```diff
@@ -104,6 +105,15 @@
     "grundr_bew_zeiten": int,
     "priv_rente_m": float,
     "schwerbeh_g": bool,
+    "rententrechtl_zeit": float,
```

Comment:
Do I gather correctly from the code below that these are all in months? Then we would need to append `_m` to all of them. Please have another look at [GEP 1](https://gettsim.readthedocs.io/en/latest/geps/gep-01.html) (no worries, I know it is impossible to follow all the guidelines outlined there until you have worked with the code for a very long time, but they are important for its long-term healty)

Reply:
the variable rentenrechtl_zeit is removed now. all remaining "zeiten" are stock variables.